src/assets/v-icon-white-thick.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="26" height="20" viewBox="0 0 26 20">
```

Comment:
WBTC-A should be right after ETH-A (not related to this file, just first comment i want to make).